tests/Form/ButtonTest.php

```
@@ -0,0 +1,852 @@
+                ->formtarget(Target::BLANK)
+                ->render(),
+            "Failed asserting that element renders correctly with 'formtarget' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(
+            Button::class,
+            ['class' => 'default-class'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)